### DISCUSSION TITLE
bot.config.superusers的顺序和.env.dev的顺序不一致

### DISCUSSION THREAD

**Discussion by wanglaoshi92** [nonebot/discussions#207] [问答] [answered]
用的真寻机器人里面有些功能是给第一位超级管理员私发信息
但是bot.config.superusers的顺序每次启动服务都不太一致导致发送混乱
如果是和nb2无关万分抱歉

**Comment by yanyongyu** [comment, marked as answer]
superuser是个set，set不存在顺序关系，你可以给真寻提issue

**Reply by wanglaoshi92** [reply]
好的 多谢 因为我之前用一个nb1框架机器人的时候也有类似的配置和代码 那个当时是没什么问题 所以以为是nb2的问题
